test/test_read.ml

```diff
@@ -12,7 +12,57 @@ let from_file () =
   Alcotest.(check Testable.yojson) __LOC__ Fixtures.json_value (Yojson.Safe.from_file input_file);
   Sys.remove input_file
 
+let unquoted_from_string () =
+  Alcotest.(check Testable.yojson)
+    __LOC__
+    Fixtures.unquoted_value
+    (Yojson.Safe.from_string Fixtures.unquoted_json)
+
+let unquoted_from_lexbuf () =
```

Comment:
This is not checking `unquoted_from_lexbuf`. Such a function would be:
```ocaml 
let unquoted_from_lexbuf () =
  let lexbuf = Lexing.from_string Fixtures.unquoted_json in
  let lexer_state = Yojson.init_lexer () in
  Alcotest.(check Testable.yojson)
    __LOC__ Fixtures.unquoted_value
    (Yojson.Safe.from_lexbuf lexer_state lexbuf)
```

Instead, this test is testing `map_ident` and `map_string`. I would rename: 
```suggestion
let map_ident_and_string () =
```